ui/src/notebooks/components/Header.tsx

```diff
@@ -0,0 +1,13 @@
+import React, {FC} from 'react'
+
+import {Page} from '@influxdata/clockface'
+
+const Header: FC<{}> = () => {
+  return (
+    <>
+      <Page.Title title="Notebooks" />
+    </>
+  )
```

Comment:
```suggestion
  return  <Page.Title title="Notebooks" />
```

Reply:
imma ignore this one as the next PR adds it right back